lib/rubocop/cop/rspec/mixin/skip_or_pending.rb

```diff
@@ -10,13 +10,18 @@ module SkipOrPending
         # @!method skipped_in_metadata?(node)
         def_node_matcher :skipped_in_metadata?, <<-PATTERN
           {
-            (send _ _ <#skip_or_pending? ...>)
-            (send _ _ ... (hash <(pair #skip_or_pending? { true str dstr }) ...>))
+            (send _ _ <#skip_or_pending_sym? ...>)
+            (send _ _ ... (hash <(pair #skip_or_pending_sym? { true str dstr }) ...>))
           }
         PATTERN
 
-        # @!method skip_or_pending?(node)
-        def_node_matcher :skip_or_pending?, '{(sym :skip) (sym :pending)}'
+        # @!method skip_or_pending_sym?(node)
+        def_node_matcher :skip_or_pending_sym?, '{(sym :skip) (sym :pending)}'
+
+        # @!method skip_or_pending_block?(node)
```

Comment:
Nice! It's probably worth mentioning in the method docs that it matches exa﻿﻿ctly, as this is non-trivial.

Example:
```ruby
        # @!method examples_inside_block?(node)
        #   Match examples defined inside a block which is not a hook
        #
        #   @example source that matches
        #     %w(r g b).each do |color|
        #       it { is_expected.to have_color(color) }
        #     end
        #
        #   @example source that does not match
        #     before do
        #       it { is_expected.to fall_into_oblivion }
        #     end
        #
        #   @param node [RuboCop::AST::Node]
        #   @return [Array<RuboCop::AST::Node>] matching nodes
        def_node_matcher :examples_inside_block?, <<~PATTERN
          (block !#{send_pattern('#Hooks.all')} _ #examples?)
        PATTERN
```

Basing on just the name I have some questions:
1. Would it match a block form of `skip` call? Probably not, right?
```
$ ruby-parse -e 'it { skip { } }'
(block
  (send nil :it)
  (args)
  (block
    (send nil :skip)
    (args) nil))
```
2. Would it really match multi-statement blocks with a `begin`? Seems unlikely, as `<` syntax only matches elements following `block`, not nested
```
$ ruby-parse -e 'd { 1; skip }'
(block
  (send nil :d)
  (args)
  (begin
    (int 1)
    (send nil :skip)))
```

Reply:
Sorry for the late. I updated this PR.